Services/SaveGameDAO.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Milestone_cst_350.Models;
+using System.Data.SqlClient;
+
+namespace Milestone_cst_350.Services
+{
+    public class SaveGameDAO
+    {
+
+        string connectionString = @"Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=Test;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
+
+        public List<SaveGameModel> AllProducts()
```

Comment:
`AllProducts` and related `_products` variables should be changed to be inline with application model naming.

Reply:
Fixed